pennylane/ops/identity.py

```diff
@@ -46,7 +46,7 @@ class Identity(CVObservable, Operation):
 
     ev_order = 1
 
-    def __init__(self, *params, wires=None, do_queue=True, id=None):
+    def __init__(self, *params, wires=None, do_queue=None, id=None):
```

Comment:
Not particularly related to this PR, but should the docstring have an "Args: ... " section? :thinking:

Reply:
Filled in the missing arguments in the docstrings: 94915cd8a1748832d3d6d4ef3b35423e7b8ecb0b
I don't think we need to have anything on `*params`, since it doesn't have any effect, right?

Reply:
Thanks for the extra mile! :)
No, `params` doesn't need a comment I think :)